cli/flox/src/utils/search.rs

```diff
@@ -80,3 +80,17 @@ pub(crate) fn construct_search_params(
     debug!("search params raw: {:?}", params);
     Ok(params)
 }
+
+/// `search` specific curated terms
+pub(crate) fn suggest_curated_package(input: &str) -> Option<&'static str> {
+    let suggestion = match input {
+        "node" => "nodejs",
+        "java" => "jdk",
+        "npm" => "nodejs",
+        "rust" => "cargo",
```

Comment:
Question: this won't get you a working Rust setup since you also need `rustc`, are we ok with that?

Reply:
cargo will give you a working rust installation as well (you can run `cargo build` and friends). You wont have `rustc` on your PATH though - but if its either `cargo` with coupled `rustc` or `rustc` without `cargo` i guerss the former is preferable?
Anyway, I'll let product decide over that.

It would also be possible to provide multiple suggestion, but I'd rather not issue multiple additional searches.